benchmarks/bench/scan/exclusive/base.cuh

```diff
@@ -25,67 +25,62 @@
  *
  ******************************************************************************/
 
+#include <look_back_helper.cuh>
 #include <cub/device/device_scan.cuh>
 
-#include <type_traits>
-
 #if !TUNE_BASE
+#if TUNE_TRANSPOSE == 0
+#define TUNE_LOAD_ALGORITHM cub::BLOCK_LOAD_DIRECT
+#define TUNE_STORE_ALGORITHM cub::BLOCK_STORE_DIRECT
+#else // TUNE_TRANSPOSE == 1
+#define TUNE_LOAD_ALGORITHM cub::BLOCK_LOAD_WARP_TRANSPOSE
+#define TUNE_STORE_ALGORITHM cub::BLOCK_STORE_WARP_TRANSPOSE
+#endif // TUNE_TRANSPOSE
+
+#if TUNE_LOAD == 0
+#define TUNE_LOAD_MODIFIER cub::LOAD_DEFAULT
+#else // TUNE_LOAD == 1
+#define TUNE_LOAD_MODIFIER cub::LOAD_CA
+#endif // TUNE_LOAD
+
 template <typename AccumT>
 struct policy_hub_t
 {
+  template <int NOMINAL_BLOCK_THREADS_4B,
+            int NOMINAL_ITEMS_PER_THREAD_4B,
+            typename ComputeT,
+            cub::BlockLoadAlgorithm LOAD_ALGORITHM,
+            cub::CacheLoadModifier LOAD_MODIFIER,
+            cub::BlockStoreAlgorithm STORE_ALGORITHM,
+            cub::BlockScanAlgorithm SCAN_ALGORITHM>
+  using agent_policy_t = cub::AgentScanPolicy<
+    NOMINAL_BLOCK_THREADS_4B,
+    NOMINAL_ITEMS_PER_THREAD_4B,
+    ComputeT,
+    LOAD_ALGORITHM,
+    LOAD_MODIFIER,
+    STORE_ALGORITHM,
+    SCAN_ALGORITHM,
+    cub::MemBoundScaling<NOMINAL_BLOCK_THREADS_4B, NOMINAL_ITEMS_PER_THREAD_4B, ComputeT>,
+    delay_constructor_t>;
+
```

Comment:
Is there any place where we use this?

In all other policies we just add the delay_constructor_t

Reply:
There are no other places like this because other places do not have `MemBoundScaling` before delay constructor. This alias ensures that the threads / items that go into scaling are the same as the ones that are passed to scan policy.